src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -1104,6 +1127,26 @@ def __call__(
 class FlaxBertForPreTraining(FlaxBertPreTrainedModel):
     module_class = FlaxBertForPreTrainingModule
 
+    def __init__(
```

Comment:
Bit confused why we need the `__init__` here but not for the previous `FlaxBertModel`? As I understood all subclasses need to implement their own `__init__`? Or can we just rely on the parent one since it's unchanged?

Reply:
Removed on the new version of the code.